airflow/bin/cli.py

```diff
@@ -597,8 +608,12 @@ class CLIFactory:
             'help': "List dag runs given a DAG id. If state option is given, it will only "
                     "search for all the dagruns with the given state. "
                     "If no_backfill option is given, it will filter out "
-                    "all backfill dagruns for given dag id",
-            'args': ('dag_id', 'no_backfill', 'state', 'output',),
+                    "all backfill dagruns for given dag id"
+                    "If exec_date_from is given, it will filter out "
+                    "all the dagruns that were executed before this date. "
+                    "If exec_date_to is given, it will filter out "
+                    "all the dagruns that were executed after this date. ",
+            'args': ('dag_id_opt', 'no_backfill', 'state', 'output', "exec_date_from", "exec_date_to"),
```

Comment:
`airflow backfill` has `--start_date` and `--end_date` args that serve a similar purpose. I think you should re-used those names instead of creating new ones.

Reply:
Done